website/source/docs/agent/config-entries/service-router.html.md

```diff
@@ -109,124 +131,128 @@ routes = [
   stops further evaluation. Traffic that fails to match any of the provided
   routes will be routed to the default service.
 
-  - `Match` `(ServiceRouteMatch: <optional>)` - A set of criteria that can
```

Comment:
Aside from splitting up the following, the changes are to just fix the indentation and reflow the remaining text. No content changes.